.github/actions/helm-dry-run/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Helm Dry Run
```

Comment:
I am not sure why you chosen `dry-run` as the name of this action, common meaning (or at least what I always used that for) is for something that get run without actually making changes or side effects happening, before the real run.

Here you are testing helm charts on a real kubernetes cluster, so to me this is more an integration/live test.

Reply:
I indeed hesitated to call it "helm-integration-tests" (since we already have "helm-unit-tests"), would that be ok?

Reply:
`helm-integration-tests` looks really reasonable to me